ci-operator/config/openshift/openshift-ansible/openshift-openshift-ansible-release-4.13.yaml

```diff
@@ -7,7 +7,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.10
+    tag: golang-1.18
```

Comment:
Is it a typo? Didn't see this change in the other file

Reply:
nice catch! I was playing with this and it slipped through by accident. thx!